modules/account/accountService.js

```diff
@@ -0,0 +1,18 @@
+const queryBulider = require('./accountQueryBuilder');
+HttpStatus = require('http-status-codes');
+
+const login = async (req, res, next) => {
+  try {
+    const logins = await queryBulider.login(req);
```

Comment:
Be more specific is using variables
The variable here can be userData which is much more undestandable